### ISSUE TITLE
Documentation should list shortcuts

### ISSUE THREAD

**Issue by blacklightpy** [bluebyt/Wayfire-dots#6]
It's hard to get started when you don't even know how to start the file manager, if when you've also replaced the original config.

**Comment by bluebyt** [comment]
See the shortcut in config file $HOME/.config/wayfire.ini

**Comment by blacklightpy** [comment]
Yes, but the key bindings are not grouped together, and to a person coming from another compositor, it'll be hard to make sense of the right keys. Some basic key bindings can be listed on the README, while asking the user to explore the rest from the configuration file.
